www/install.sh

```diff
@@ -171,7 +193,7 @@ if [ -e "$dest/just" ] && [ "$force" = false ]; then
   err "\`$dest/just\` already exists"
 else
   mkdir -p "$dest"
-  install -m 755 "$td/just" "$dest"
+  install_just "$td/just" "$dest"
```

Comment:
I don't think there's any particular advantage to using `install` when it's available, so let's just use `cp` and `chmod` directly.